ToopherSecondFactor/toopher-openam.js

```diff
@@ -1,183 +1,84 @@
 // vim: ts=2 sw=2 expandtab cindent
```

Comment:
it's a bit of a stretch to say that a vim modeline unacceptably complects the code.